doc/api/next_api_changes/deprecations.rst

```diff
@@ -415,3 +415,9 @@ parent figure, though they can be later removed with ``ax.remove()``.
 
 ``.BboxBase.inverse_transformed`` is deprecated (call `.BboxBase.transformed`
 on the `~.Transform.inverted()` transform instead).
+
+``pie(normalize=None)``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
Add *normalize*  keyword argument to ``Axes.pie``
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

This needs to be understandable to someone who is reading this page with no context.

I would have put this in the "behavior change" file, not the deprecation file as we are adding, not removing, functionality.